README.md

```diff
@@ -48,6 +49,8 @@ Before running the app locally, you have to:
 2. Create the [backend/.env](backend/.env) file based on
    [backend/.env.example](backend/.env.example) and set the variables with the values
    from the previous step.
+3. Generate RSA key pair (for example on https://cryptotools.net/rsagen) and copy
```

Comment:
I'm wondering if we still here in the README should add a new `1.` step saying you should create a group for the app and then add a new last step (`5.`) to say that IF you want to try out the consent stuff, then you should host the party-configuration somewhere and configure the group to use that domain and also configure `CONSENT_REQUEST_TOKEN_ISSUER` to `https://` + `party_configuration_domain`.